google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -266,9 +357,29 @@ public Builder toBuilder() {
     return new Builder(this);
   }
 
-  /** Returns the map of bindings that comprises the policy. */
+  /**
+   * Returns the map of bindings that comprises the policy.
+   *
+   * @throws IllegalArgumentException if policy version is equal to 3 or has conditional bindings.
+   */
   public Map<Role, Set<Identity>> getBindings() {
-    return bindings;
+    checkArgument(
+        !isConditional(this.version, this.bindingsList),
+        "getBindings() is only supported with version 1 policies and non-conditional policies");
+    ImmutableMap.Builder<Role, Set<Identity>> bindingsV1Builder = ImmutableMap.builder();
```

Comment:
You can use an `ImmutableSetMultimap` instead, regardless of whether you change the method return type. You can always get the map out with `Multimaps.asMap()`.